inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java

```diff
@@ -112,7 +112,7 @@ public Boolean reportSnapshot(TaskSnapshotRequest request) {
     @Override
     @Transactional(rollbackFor = Throwable.class, isolation = Isolation.REPEATABLE_READ)
```

Comment:
why not to modify `Isolation.REPEATABLE_READ` to `Isolation.READ_COMMITTED`

Reply:
Because the inner method `getTaskResult` needs `Isolation.READ_COMMITTED`.